### ISSUE TITLE
Script fails with misleading message if python command is not available

### ISSUE THREAD

**Issue by HoT-1** [python-virtualenvwrapper/virtualenvwrapper#54]
If the command python is not available the following code fails to detect the location of python and continues with VIRTUALENVWRAPPER_PYTHON=''.

``` bash
# Locate the global Python where virtualenvwrapper is installed.
if [ "${VIRTUALENVWRAPPER_PYTHON:-}" = "" ]
then
    VIRTUALENVWRAPPER_PYTHON="$(command \which python)"
fi
```

The result is the following output:
```
Command '' not found.
If Python could not import the module virtualenvwrapper.hook_loader,
check that virtualenvwrapper has been installed for
VIRTUALENVWRAPPER_PYTHON= and that PATH is
set properly.
```

The issue can occur even if python is installed. In my case python3 was installed in WSL Ubuntu 22.04. But per default there is no link from python to python3. For this link to exists the package python-is-python3 must be installed.

**Comment by dhellmann** [comment]
This should be resolved by #28. Could you see if version [6.0.0.0a2](https://github.com/python-virtualenvwrapper/virtualenvwrapper/releases/tag/6.0.0.0a2) still has the same behavior?

**Comment by dhellmann** [comment]
Closing as fixed.
